README.md

```diff
@@ -375,6 +375,7 @@ These can be useful for osint and social engineering.
 - [Infringement Report](https://infringement.report/) - The web's best image copyright infringement search tool
 - [Tineye](https://tineye.com/) - Image search and recognition company
 - [Flickr](https://flickr.com/search/) - Home to tens of billions of photos and 2 million groups
+- [Jimpl](https://jimpl.com/) - Online EXIF data viewer
```

Comment:
The proper description should be `Online photo metadata and EXIF data viewer` (this is what I get if I google Jimpl)

Reply:
@edoardottt done